Makefile

```diff
@@ -0,0 +1,20 @@
+PREFIX = /usr/
+
+all: install
+
+install:
```

Comment:
This rule depends on a couple of files, so they should be listed as dependencies.